### PR TITLE
Bids 3142/group filter for slot viz

### PR DESCRIPTION
**Pull request by MauserBitfly**
This PR:
- fixes the translation of the 'visible' icon in the slot filters
- refactors the slot viewer to handle more complex headers (while keeping the SlotVizViewer as dumb as encapsulated as possible) 
- adapts / implements the PrimeVue MulitSelcect (.p-multiselect) styles according to our styles
  - Please note that the Total Group is separated from the rest of the List, this is not in the Layouts but was cross checked with the designer.
- adds network label to to the slot viz header
- adds an `?` icon to the header leading to the docu

### CODE REVIEW

File: frontend/components/slot/viz/SlotVizViewer.vue [hunk 1]
```diff
@@ -95,63 +101,122 @@ watch(props, () => {
...
-    <div class="rows">
-      <div class="row">
+    <div class="header-row">
+      <BcTooltip class="info" :text="$t('slotViz.info_tootlip')" :dont-open-permanently="true">
+        <BcLink to="https://kb.beaconcha.in/v2beta/slot-visualization#how-does-it-work" target="_blank" class="link">
+          <FontAwesomeIcon :icon="faInfoCircle" />
+        </BcLink>
+      </BCTooltip>
+      <div class="filter-row">
         <BcToggleMultiBar v-model="selectedCategories" :icons="icons" />
```
Comment (D13ce) [inline frontend/components/slot/viz/SlotVizViewer.vue:111]: **Question:** The tooltips on mobile are tedious as they can only be viewed by pressing the button. But pressing the button also changes its toggle state so it is impossible to see the tooltip without performing an action.
We might want to address this issue with the product owner, right?

The same applies to the info icon to the left of these buttons of course.

File: frontend/assets/css/prime.scss [hunk 2]
```diff
@@ -294,83 +295,76 @@
...
+    float: left;
+    margin-right: var(--padding);
+    padding-left: var(--padding-small);
+    .p-multiselect-close {
+      display: none;
+    }
+  }
+
+  .p-multiselect-items-wrapper {
+    max-width: 200px;
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/assets/css/prime.scss:345]: I want to have a  reasonable default for the dialog width - and I think 200 is good.
If a specific component needs more or less he could style the dialog with
```
      <MultiSelect
        :panel-props="{
          class:'my-fucking-panel-class'
        }"
      >
```

File: frontend/components/dashboard/ValidatorSlotViz.vue
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/components/dashboard/ValidatorSlotViz.vue:1]: sure makes sense to sort them

[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/components/dashboard/ValidatorSlotViz.vue:1]: I think like it that way. If users really get confused we can adapt later.

[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/components/dashboard/ValidatorSlotViz.vue:1]: reduced the break point for setting the mobile width of the dropdown

File: frontend/components/slot/viz/SlotVizViewer.vue (hunk 1 above)
Comment (MauserBitfly) [reply frontend/components/slot/viz/SlotVizViewer.vue:111]: That's a general problem of tooltips on interactive items ... I mean we could prevent the action on the first click and only show the tooltip, but how does the user know he can click again? 

Tooltips are a bad UX for mobile and the UI should be intuitive enough without them and not rely on them. 

And that could be a general discussion and not for one specific instance ;-)

File: frontend/i18n/en.json [hunk 3]
```diff
@@ -150,12 +150,13 @@
...
   "slotViz": {
     "head": "Head",
+    "info_tootlip": "Slot viz explainer",
     "filter": {
       "attestation": "If enabled, attestations will be shown.",
       "proposal": "If enabled, blocks will be shown.",
       "slashing": "If enabled, slashings will be shown.",
       "sync": "If enabled, sync committees will be shown.",
-      "visible": "If enabled, all duties will be shown."
+      "visible": "If enabled, all duty icons for the duties selected to the left will be shown."
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/i18n/en.json:159]: added a max width for tooltips so they don't get out of hand

File: frontend/components/dashboard/ValidatorSlotViz.vue
[the remark this answers is not in the document]
Comment (D13ce) [reply frontend/components/dashboard/ValidatorSlotViz.vue:1]: Sorting does not work. Please note how the order of the groups in top row depends on the order of the group selection in the drop-down: [Screencast](https://github.com/gobitfly/beaconchain/assets/114066135/2d001ac5-9a5c-40ae-9248-de021ee2a736)

[the remark this answers is not in the document]
Comment (D13ce) [reply frontend/components/dashboard/ValidatorSlotViz.vue:1]: Fixed with [52729a9](https://github.com/gobitfly/beaconchain/pull/496/commits/52729a9ea33064165f2451b7016fa989be287ea6)

**Review by D13ce — APPROVED** [review]
LGTM

(Please note that I have added a small commit to improve a tooltip text as discussed)

**Pull request merged**